api/Worker.json

```diff
@@ -542,18 +532,11 @@
               "version_added": "10",
               "notes": "Internet Explorer does not support `Transferable` objects."
             },
-            "nodejs": [
-              {
-                "version_added": "12.17.0",
-                "partial_implementation": true,
-                "notes": "Supports `transferList` argument for transferring `ArrayBuffer` and `MessagePort` objects"
-              },
-              {
-                "version_added": "11.7.0",
-                "partial_implementation": true,
-                "notes": "Supports `transferList` argument for transferring `ArrayBuffer` and `MessagePort` objects"
-              }
-            ],
+            "nodejs": {
+              "version_added": "11.7.0",
+              "partial_implementation": true,
+              "notes": "Only support `transferList` parameter as an optional array of objects."
```

Comment:
This is not as accurate as the original version though. I think the most accurate way to refer the supported types is https://nodejs.org/api/worker_threads.html#portpostmessagevalue-transferlist.

Reply:
How about:

> Only support passing the second parameter as an optional array of objects, not support passing the second parameter as an options object.
> Only support transferring `ArrayBuffer` and `MessagePort` objects.